Microsoft.Toolkit.Uwp.UI.Controls.Core/TileControl/TileControl.cs

```diff
@@ -173,17 +173,21 @@ private async Task<bool> LoadImageBrushAsync(Uri uri)
                 var loadCompletedSource = new TaskCompletionSource<bool>();
                 _brushVisual = compositor.CreateSurfaceBrush(_imageSurface);
 
-                _imageSurface.LoadCompleted += (s, e) =>
+                void LoadCompleted(LoadedImageSurface sender, LoadedImageSourceLoadCompletedEventArgs args)
                 {
-                    if (e.Status == LoadedImageSourceLoadStatus.Success)
+                    _imageSurface.LoadCompleted -= LoadCompleted;
```

Comment:
We could just reuse the sender instance to avoid capturing the surface as well, since it's the same one? 🤔
```suggestion
                    sender.LoadCompleted -= LoadCompleted;
```